media-libs/opencolorio/opencolorio-2.1.2.ebuild

```diff
@@ -55,6 +55,7 @@ RESTRICT="test"
 
 PATCHES=(
 	"${FILESDIR}"/${PN}-2.1.1-gcc12.patch
+	"${FILESDIR}"/${PN}-2.1.2-musl-stol.patch
```

Comment:
I think this is typo: `stol` -> `strtol`.

Reply:
Yep right, I'll fix it